bitnami/postgresql-ha/templates/postgresql/witness-statefulset.yaml

```diff
@@ -625,6 +625,5 @@ spec:
         {{- if .Values.persistence.selector }}
         selector: {{- include "common.tplvalues.render" (dict "value" .Values.persistence.selector "context" $) | nindent 10 }}
         {{- end -}}
-        {{- include "common.storage.class" (dict "persistence" .Values.persistence "global" .Values.global) | nindent 8 }}
```

Comment:
Same here, I'd remove the lines 619-621: https://github.com/bitnami/charts/blob/e909092a93db4ecd386aa88439fd98cc7887eb79/bitnami/postgresql-ha/templates/postgresql/witness-statefulset.yaml#L619-L621